docs/src/main/sphinx/admin/fault-tolerant-execution.rst

```diff
@@ -294,12 +328,16 @@ for your storage solution.
      - Description
      - Default value
    * - ``exchange.base-directories``
-     - The base directory URI location that the exchange manager uses to store
-       spooling data. Only supports S3 and local filesystems.
+     - Comma-separated list of URI locations that the exchange manager uses to
+       store spooling data. Only supports S3 and local filesystems.
      -
    * - ``exchange.encryption-enabled``
      - Enable encrypting of spooling data.
      - ``true``
+   * - ``exchange.compression-enabled``
+     - Enable compression of spooling data. Setting to ``true`` is recommended
```

Comment:
Generally it is recommended to enable it when using `FileSystemExchange` (with any underlying storage)